fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -307,7 +309,8 @@ public List<ShardGroupInfo> listShardGroup() {
         }
     }
 
-    public List<Long> createShards(int numShards, int replicaNum, FilePathInfo pathInfo, FileCacheInfo cacheInfo, long groupId)
+    public List<Long> createShards(int numShards, int replicaNum, FilePathInfo pathInfo, FileCacheInfo cacheInfo, long groupId,
```

Comment:
how about add a new method createShards() without the matchShardIds parameter which is preferred in most cases. So most calls will still go without the `matchShardIds` parameter and won't have to pass a `null`  for it.

Reply:
done